### ISSUE TITLE
error: use of undeclared identifier 'set_unexpected'

### ISSUE THREAD

**Issue by ryandesign** [Edrusb/DAR#52]
dar 2.7.10 fails to build with clang 16:

https://build.macports.org/builders/ports-10.9_x86_64-builder/builds/236560/steps/install-port/logs/stdio

```
erreurs.cpp:163:9: error: use of undeclared identifier 'set_unexpected'
        set_unexpected(inattendue);
        ^
```

My understanding is that clang 16 and later builds in C++17 mode by default and that `std::set_unexpected` was removed in C++17.

Manually adding `-std=c++14` to the `CXXFLAGS` environment variable works around the problem.

**Comment by Edrusb** [comment]
Thanks Ryan for this feedback. 

Yes, it makes sense removing this as dar/libdar no more use exception specifications for a long time now... set_unexepected() is thus completely useless in dar/libdar

**Comment by Edrusb** [comment]
set_unexpected() is now removed (git/branch_2.7.x)  and will be available with next release (2.7.11)

**Comment by Edrusb** [comment]
closing this issue with release 2.7.11 which include this fix
